modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/processor/OperationMessageProcessor.java

```diff
@@ -167,4 +182,44 @@ public void setMuleContext(MuleContext muleContext)
         this.muleContext = muleContext;
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public MetadataResult<List<MetadataKey>> getMetadataKeys() throws MetadataResolvingException
+    {
+        return metadataMediator.getMetadataKeys(getMetadataContext());
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public MetadataResult<OperationMetadataDescriptor> getMetadata() throws MetadataResolvingException
+    {
+        return metadataMediator.getMetadata();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public MetadataResult<OperationMetadataDescriptor> getMetadata(MetadataKey key) throws MetadataResolvingException
+    {
+        return metadataMediator.getMetadata(getMetadataContext(), key);
+    }
+
+    private MetadataContext getMetadataContext() throws MetadataResolvingException
+    {
+        //FIXME bug when the configuration is accessed using the extensionModel, but the implicit config is dynamic
```

Comment:
change for a TODO and point to a jira issue... also consider that we're not even sure if we want to support dynamic configs to begin with